src/app/shared/components/search-input/search-input.component.ts

```diff
@@ -10,9 +10,17 @@ import * as fromRoot from '@/app/store';
 @Component({
   selector: 'tmm-search-input',
   templateUrl: './search-input.component.html',
-  styleUrls: ['./search-input.component.scss']
+  styleUrls: ['./search-input.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => SearchInputComponent),
+      multi: true
+    }
+  ]
 })
-export class SearchInputComponent implements OnInit, OnDestroy {
+export class SearchInputComponent implements OnInit, OnDestroy, ControlValueAccessor {
```

Comment:
Implement the control value accessor so we can treat this component like a normal form input.